modules/preview-react/InformationHighlight/stories/visual_testing/testing.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import {
+  StaticStates,
+  ComponentStatesTable,
+  permutateProps,
+} from '@workday/canvas-kit-react/testing';
+import {withSnapshotsEnabled} from '../../../../../utils/storybook';
+
+import {InformationHighlight} from '@workday/canvas-kit-preview-react/InformationHighlight';
+
+export default {
+  title: 'Testing/Preview/Information Highlight',
+  component: InformationHighlight,
+};
```

Comment:
Need to add this section as this allows Chromatic to take visual snapshots. It defaults to `false`. 

```suggestion
export default {
  title: 'Testing/Preview/Information Highlight',
  component: InformationHighlight,
    parameters: {
    chromatic: {
      disable: false,
    },
  },
};
```

Reply:
ah! good to know. updated :)